pkg/subctl/cmd/deploybroker.go

```diff
@@ -52,6 +52,9 @@ var validComponents = []string{components.ServiceDiscovery, components.Connectiv
 func init() {
 	deployBroker.PersistentFlags().BoolVar(&globalnetEnable, "globalnet", false,
 		"enable support for Overlapping CIDRs in connecting clusters (default disabled)")
+
+	_ = deployBroker.PersistentFlags().MarkDeprecated("globalnet", "please use --components instead")
```

Comment:
Shall we also give the proper usage?
```suggestion
	_ = deployBroker.PersistentFlags().MarkDeprecated("globalnet", "please use --components globalnet")
```

Reply:
But, will that work? don't you also need to add all the other components too? (is --components additive?)

so probably it should be --components service-discovery,globalnet,operator ? ?? 

@skitt what would it be exactly ^?

Reply:
I don't know I start to feel that we should keep this flag, so the --components thing is internal design, and --globalnet is part of the UI to make it easier?

Reply:
I need to check but I think there are some edge-cases that don’t work too well with `--components globalnet` v. `--globalnet`. When I started working on full componentisation, it quickly became apparent too that Globalnet isn’t a component in the same way as connectivity and service discovery; it’s rather a setting which affects both... I’m not sure what the best way would be to represent it in the flags.